### PR TITLE
Remove logrus.Infof on bindings start and attach

### PR DESCRIPTION
**Pull request by rhatdan**
We are only doing logrus.Info on a couple of bindings. I see no reason
to only do this on a couple, seems like debugging information more then
useful to the caller. We had a complaint about this so removing.

Fixes: https://github.com/containers/podman/issues/13504

[NO NEW TESTS NEEDED]

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

#### Does this PR introduce a user-facing change?
None

```release-note
None
```

### CODE REVIEW

**Comment by mheon** [comment]
LGTM

**Comment by rhatdan** [comment]
The other two look like they could be warnings, but I am not sure why they are marked as info.  Perhaps they happen too often.

**Comment by rhatdan** [comment]
@mheon WDYT on https://github.com/containers/podman/blob/main/pkg/bindings/containers/attach.go#L57 Should we leave this change it to Warn or Debug?

**Comment by mheon** [comment]
I would just remove it entirely.

File: cmd/podman/system/info.go [hunk 1]
```diff
@@ -11,8 +11,8 @@ import (
 	"github.com/containers/podman/v4/cmd/podman/registry"
 	"github.com/containers/podman/v4/cmd/podman/validate"
 	"github.com/containers/podman/v4/libpod/define"
-	"github.com/ghodss/yaml"
 	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v3"
```
Comment (Luap99) [inline cmd/podman/system/info.go:15]: This is not related, please remove it.

**Comment by mheon** [comment]
Ehhhh. I don't really see that mattering much... But if we want to keep it, Debug is definitely the way to go. The container will still function (mostly) normally even without a successful resize.
